documentation/asciidoc/computers/os/graphics-utilities.adoc

```diff
@@ -138,7 +138,7 @@ Returns the throttled state of the system. This is a bit pattern - a bit being s
 
 ==== measure_temp
 
-Returns the temperature of the SoC as measured by the on-board temperature sensor.
+Returns the temperature of the SoC as measured by its internal temperature sensor. On Raspberry Pi 4 the `pmic` subcommand can be used to instead return the temperature of the PMIC.
```

Comment:
It's not entirely clear if this means `vcgencmd pmic` or `vcgencmd measure_temp pmic`?
Also, I may just be reading it wrong but the "instead" implies that the former option isn't available and only the latter option is available? Maybe it's worth rewording to use "also" to clarify that both options are available, IYSWIM?

Reply:
I've changed the wording slightly. 

(A quick test on a Pi 3B+ and Pi 1 reveals that on pre-Pi4 boards, `vcgencmd measure_temp pmic` will silently ignore the `pmic` parameter and just return the temperature of the SoC).